cpp/src/tests/centrality/katz_centrality_test.cu

```diff
@@ -0,0 +1,131 @@
+#include "gtest/gtest.h"
+#include "gmock/gmock.h"
+#include "gmock/gmock-generated-matchers.h"
+#include "high_res_clock.h"
+#include "cuda_profiler_api.h"
+#include <cugraph.h>
+#include "test_utils.h"
+#include <thrust/device_ptr.h>
+#include <fstream>
+
+std::vector<int>
+getGoldenTopKIds(std::string file, int k = 10) {
+  std::vector<int> vec;
+  std::ifstream fin(file);
+  int val;
+  while (fin>>val) {
+    vec.push_back(val);
+  }
+  vec.resize(k);
```

Comment:
Why not just read first k?

Reply:
fin.close() missing here?

Reply:
> fin.close() missing here?

Destructor of `std::basic_ifstream` calls the close function.

Reply:
Fixed